src/lib/auxiliary_database/filtered_external_transition.ml

```diff
@@ -94,10 +94,13 @@ let of_transition tracked_participants external_transition =
         Coda_state.Protocol_state.blockchain_state
         @@ protocol_state external_transition }
   in
+  (* real state body hash is used only for pending coinbases *)
+  let state_body_hash = State_body_hash.dummy in
```

Comment:
These shouldn't be dummy. I think you can determine the hash of `Protocol_state.body (External_transition.Validated.protocol_state parent.validated_transition)` from where this function is called